src/Chapter2.hs

```diff
@@ -842,7 +857,9 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [a] -> [a]
+rotate x _ | x < 0 = []
```

Comment:
Guards are better when used with several cases, that would make it much more readable, also you can reuse the variables.